services/sleeper/validation/input/inputs.json

```diff
@@ -2,5 +2,6 @@
     "input_1": "single_number.txt",
     "input_2": 2,
     "input_3": false,
-    "input_4": 4
-}
+    "input_4": 4,
+    "input_5": 1024
+}
```

Comment:
Here again

Reply:
well here my linter removes the empty line